user/serializers.py

```diff
@@ -22,3 +24,38 @@ def update(self, instance, validated_data):
             user.save()
 
         return user
+
+
+class AuthTokenSerializer(serializers.Serializer):
+    email = serializers.EmailField(
+        label=_("Email"),
+        style={"input_type": "email"}
+    )
+    password = serializers.CharField(
+        label=_("Password"),
+        style={"input_type": "password"}
+    )
+
+    def validate(self, attrs):
+        email = attrs.get("email")
+        password = attrs.get("password")
+
+        if email and password:
+            user = authenticate(email=email, password=password)
+
+            if user:
+                if not user.is_active:
+                    msg = _("User account is disabled.")
+                    raise serializers.ValidationError(
+                        msg,
+                        code="authorization"
+                    )
```

Comment:
`django.contib.auth.authenticate` won't authenticate disabled users, so this check is redundant.